data/aaronmack.json

```
@@ -135,12 +127,12 @@
       "url": "https://server10.xyzzyxwz.top:12030"
     },
     {
-      "title": "Wiki.js",
+      "title": "Bookstack",
       "date": "Continuing 2023",
       "icon": "FaExternalLinkSquareAlt",
       "color": "grey",
-      "description": "Wiki App (Github Fork)",
-      "url": "https://server11.xyzzyxwz.top:12030"
-    }
+      "description": "A self-built Wiki system (Github Fork)",
+      "url": "https://server15.xyzzyxwz.top:12030"
+    },
```

Kindly remove this comma at the end, it's causing the JSON check to fail. 
```suggestion
    }
```